unit/java_bytecode/java_bytecode_parse_generics/parse_generic_class.cpp

```diff
@@ -16,307 +18,148 @@
 #include <util/language.h>
 #include <util/message.h>
 #include <java_bytecode/java_bytecode_language.h>
+#include <iostream>
 
 SCENARIO(
-  "java_bytecode_parse_generics",
+  "java_bytecode_parse_generic_class_one_param",
```

Comment:
I wonder whether if we have changed the name of the scenario, whether we should also change the filename? I know it's pretty easy to do a all-project search for the scenario name, but it might make sense if all the unit test files are named to match their scenario name?